### ISSUE TITLE
Animation

### ISSUE THREAD

**Issue by crisward** [Luegg/angularjs-scroll-glue#16]
When I use a max-height css animation on a ng-repeat, the scroll-glue only scrolls to the height at the the first point in the animation. 

It would be nice if this directive could listen for the animation ending, then fire. Or perhaps fire at beginning and end.

**Comment by onbjerg** [comment]
Yes! This is exactly the scenario I am in. I have to disable the sweet animations in my project for now.

**Comment by chrismbarr** [comment]
+1 for this fix - animations seem to trip this up and right now I cannot use them AND keep my element scrolled correctly.

**Comment by toinouu** [comment]
+1

**Comment by oliversalzburg** [comment]
Is this still an issue? Could anyone provide an example repo/fiddle to repro the problem?
